frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -782,6 +782,157 @@ export function GetEnvironmentVariables(props: EnvironmentVariablesProps) {
       }
     };
 
+    const processValueFromFieldRef = (name: string, fieldRef: { fieldPath: string }) => {
+      let value = '';
+      let isError = false;
+
+      try {
+        // Support for predefined field paths
+        switch (fieldRef.fieldPath) {
+          case 'metadata.name':
+            value = pod.metadata.name || '';
+            break;
+          case 'metadata.namespace':
+            value = pod.metadata.namespace || '';
+            break;
+          case 'spec.nodeName':
+            value = pod.spec.nodeName || '';
+            break;
+          case 'spec.serviceAccountName':
+            value = pod.spec.serviceAccountName || '';
+            break;
+          case 'status.hostIP':
+            value = pod.status.hostIP || '';
+            break;
+          case 'status.podIP':
+            value = pod.status.podIP || '';
+            break;
```

Comment:
Looks like this could be automated by checking whether we can split by `.` and checking 1st and 2nd split results from pod.
But probably a better solution is to use JSONPath to try to match the fieldPath.